code/client/munkilib/munkicommon.py

```diff
@@ -1208,7 +1208,8 @@ def pref(pref_name):
         'SuppressUserNotification': False,
         'SuppressAutoInstall': False,
         'SuppressStopButtonOnInstall': False,
-        'PackageVerificationMode': 'hash'
+        'PackageVerificationMode': 'hash',
+        'AllowInsecureConnections': True
```

Comment:
I do think that this should be changed to false in a future release with ample warning.